nupic/encoders/scalar.capnp

```diff
@@ -0,0 +1,20 @@
+@0xfa7d16f86048a6e4;
+
+struct ScalarEncoderProto {
+  w @0 :Int32;
+  minval @1 :Float32;
+  maxval @2 :Float32;
+  periodic @3 :Bool;
+  n @4 :Int32;
+  radius @5 :Float32;
+  resolution @6 :Float32;
```

Comment:
Ok it actually looks like it is n, radius, and resolution that are redundant. In `initEncoder` it checks which is specified and sets the others based on that. So just pick one of them (I suggest `n`) to store and then when you load you can call `initEncoder` to calculate the other two